### DISCUSSION TITLE
Staff Namer

### DISCUSSION THREAD

**Discussion by jjpb44** [jcristy/RideNamer#21] [Ideas]
Hello! 
Do you see it viable to implement a similar function (or another plugin) that does the same thing as this plugin but to add a name to the new staff members? I think it should be easy and basically just add the format firstName secondName [Staff Type] # to the name field.
example: James Smith Handyman 10
and also be optional to have the staff name or Names in the format. So, it could be like always,
  [staff type] #
, or just the name,
"James Smith"
or both.

**Comment by jcristy** [comment]
That is most likely feasible (I'd have to check the openRCT2 API, but I'd be shocked if it weren't) and while it doesn't really fit the title, I think it fits the spirit (and the logic already implemented) of the plugin. Unfortunately, I've had about 0 time for any personal projects in the last few months. Hopefully that will change soon.

FWIW, I also have an idea for a plugin which automates hiring staff (I find it tedious to keep track of hiring more mechanics and handymen as a park grows). It'd be something like have X handymen per Y guests, S mechanics per R rides. That might also be a good home for giving them not boring names.

Thanks for the idea!
